src/observer.ts

```diff
@@ -82,10 +72,10 @@ export interface Observer<T> {
 }
 
 export interface Observable<T> {
```

Comment:
Can we verify somehow that
1. an Rx `Observable` can be passed to Ix `from()`
2. the `Observable` returned by `toObservable` can be passed to Rx methods (at the very least `from()`)

Reply:
you mean like these tests here? https://github.com/ReactiveX/IxJS/blob/ead5b475bde5e242cdaf834ed9486c20fe003876/spec/asynciterable-operators/toobservable-spec.ts#L132